src/middlewared/middlewared/etc_files/local/users.oath

```diff
@@ -0,0 +1,11 @@
+<%
+	from middlewared.utils.exceptions import DeprecatedFile
```

Comment:
Can we add this exception to the context of all templates like we do with `middleware` object?